src/glmfit.jl

```diff
@@ -68,8 +55,8 @@ function updateμ!{T<:FPVector,D,L}(r::GlmResp{T,D,L}, linPr::T)
     isempty(r.offset) ? copy!(r.eta, linPr) : broadcast!(+, r.eta, linPr, r.offset)
     updateμ!(r)
     if !isempty(r.wts)
-        r.devresid .*= r.wts
-        r.wrkwt .*= r.wts
+        map!(*, r.devresid, r.devresid, r.wts)
+	map!(*, r.wrkwt, r.wrkwt, r.wts)
```

Comment:
This one line is indented with a tab, whereas the surrounding code is indented by multiples of 4 spaces. If you could change it to spaces for consistency that would be great.